github/projects.go

```diff
@@ -466,3 +466,127 @@ func (s *ProjectsService) MoveProjectCard(ctx context.Context, cardID int64, opt
 
 	return s.client.Do(ctx, req, nil)
 }
+
+type ProjectCollaboratorOptions struct {
+	// Permission specifies the permission to grant to the collaborator.
+	// Possible values are:
+	//     read - can read, but not write to or administer this project.
+	//     write - can read and write, but not administer this project.
+	//     admin - can read, write and administer this project.
+	//
+	// Default value "write"
```

Comment:
Please choose one of the following, to make the godocs read better:

`// Default value is "write".`
`// Default value: "write".`

Either way, please end the line with a period.